pkg/distro/fedora/distro.go

```diff
@@ -442,6 +442,8 @@ var (
 				// Overwrite the default Grub2 timeout value.
 				Timeout: 5,
 			},
+			// Unset hostname so it isn't written (necessary for systemd-firstboot).
+			Hostname: common.ToPtr(""),
```

Comment:
Wouldn't it be more correct to set it to `nil` (or keep it unset)?

Reply:
Setting it to `nil` does the wrong thing when the merge between image configs gets done; it then gets turned into the distro default.